app/root.tsx

```diff
@@ -24,7 +24,7 @@ export default function App() {
 				<Meta />
 				<Links />
 			</head>
-			<body className="h-full min-h-full overflow-y-scroll bg-body">
+			<body className="h-full min-h-full overflow-y-scroll bg-base">
```

Comment:
This results in a lighter background in the docs. I think I'd like to maintain that so maybe we don't use the tokens in this context, or we make those value match what's happening in docs.

Left is this branch, right is production.
<img width="1281" alt="image" src="https://github.com/ngrok-oss/mantle/assets/1369864/32a314de-055e-4dcc-9908-5d60eb8740af">

Reply:
hmm, that previous value must've been a previous token value from Shad? Those are insanely close on my screen! hard to tell with the window shadow